.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 # Configuration file for `pre-commit` tool.
 # Source: https://pre-commit.com/#pre-commit-configyaml---top-level
-exclude: "docs|node_modules|migrations|.git|.tox|.md|sample_setups/(external-setups|jsons)|tests/deprecated"
+exclude: "docs|node_modules|migrations|.git|.tox|README.md|sample_setups/(external-setups|jsons)|tests/deprecated"
```

Comment:
@lee1043  and @acordonez, I isolated the issue of `pre-commit` skipping files to the `.md` substring in the `exclude` config. I'm not sure if `pre-commit` can handle file extensions or not (`.tox` is a not a file extension). I tried quickly Googling but didn't find any results.

I updated `.md` to `README.md` and now `pre-commit` runs locally and on CI/CD. Now `pre-commit run --all-files` picks up some issues that need to be fixed.

Let me know if `pre-commit` is working for you locally as well.

Reply:
@tomvothecoder this is also working locally for me now when I run `pre-commit run --all-files`. Thanks for looking into this!

Reply:
@tomvothecoder thank you very much for figuring this out! At some point I added .md assuming it was for file extensions because I was annoyed by pre-commit complaining for extra space at the end of line in .md files. My bad. I appreciate you figured this out!

Reply:
@acordonez and @lee1043 great and no problem!

Feel free to merge.  You can rebase your dev branches on the latest `main` to get `pre-commit` working again.